src/ol/tile.js

```diff
@@ -4,6 +4,7 @@ goog.require('ol');
 goog.require('ol.events.EventTarget');
 goog.require('ol.events.EventType');
 
+var lastSequenceNumber = 0;
```

Comment:
Yes, I think it will. That's why it should probably be a static (private) variable - I simply forgot to change that before submitting the pull request.